internal/tofu/node_resource_validate.go

```diff
@@ -281,8 +281,13 @@ var connectionBlockSupersetSchema = &configschema.Block{
 
 func (n *NodeValidatableResource) validateResource(ctx EvalContext) tfdiags.Diagnostics {
 	var diags tfdiags.Diagnostics
+	// TODO Ronny fix
```

Comment:
I still didn't fix this area. I believe I'll have to loop over all the potential providers and ensure I validate each one separately.

Reply:
NVM it is fixed